src/crypto_helpers.rs

```diff
@@ -1,15 +1,54 @@
 use nanos_sdk::bindings::*;
 use nanos_sdk::ecc::{CurvesId, DEREncodedECDSASignature};
 
-// TODO: make a macro that derives the path by itself
-// ex: make_bip32_path!(b"44'/535348'/0'/0/0";
-// =>
-pub const BIP32_PATH: [u32; 5] = [
-        0x8000002cu32,
-        0x80082b34, 
-        0x80000000, 
-        0x00000000, 
-        0x00000000];
+/// ASCII value of the character 0.
+const ZERO: u8 = 48;
+/// ASCII value of the character 9.
+const NINE: u8 = 57;
+
+/// Helper macro that creates an array from the ASCII values of a correctly formatted derivation path.
+/// Format expected: `b"44'/coin_type'/account'/change/address"`.
+///
+/// # Panics
+///
+/// Panics if the parameter does not follow the correct format.
+macro_rules! make_bip32_path {
```

Comment:
Is this bip32? We're starting with `44` so I believe it's BIP44? I'm no expert on the topic, so let me know if I completely messed that up, or if we should change the name of this macro...